### ISSUE TITLE
Penny replies with commands when mentioned or direct messaged

### ISSUE THREAD

**Issue by ericbastarache** [the-heap/penny#10]
- [ ] When you mention Penny in the #draw_it channel or direct message it directly, it should give you a list of its commands.

**Comment by teesloane** [comment]
Alright, so we have Penny responding now when we `@` mention the bot.

I suggest that we set up a really simple canned response; something like :

```
Hi I'm Penny; I can do the below if you `@` mention me and say the following!
1. @penny give prompt
2. @penny, submit prompt "<your prompt here>"

Make sure you type the above commands correctly or I won't listen to you.
```

doesn't have to be quite like that, but that might be a start!

**Comment by teesloane** [comment]
@ericbastarache I think we've gotten to a point where we can close this.  

we'll need to keep updating as we go along, but the base functionality is pretty much there. What do you think?

**Comment by ericbastarache** [comment]
I agree; and getting a list of commands from direct messaging still needs to be added as well. I think with the base functionality in place though, this shouldn't be too difficult to implement
